libethash-cl/CLMiner.cpp

```diff
@@ -265,8 +265,10 @@ CLMiner::CLMiner(unsigned _index) : Miner("cl-", _index) {}
 
 CLMiner::~CLMiner()
 {
+    DEV_BUILD_LOG_PROGRAMFLOW(cudalog, "cuda-" << m_index << " CLMiner::~CLMiner() begin");
```

Comment:
This is CLMiner ... but you log "cuda-"

Reply:
Dammed! 
Added  logging in CLMiner to this PR as I saw in PI 1598 I did it only for CUDA.
Copy&Paste Error! Will resolve! THX.